lambda/health_package/components/generic_helper.py

```diff
@@ -157,6 +157,12 @@ def get_metric_threshold(cls, metric, rule):
 
         return threshold
 
+    @classmethod
+    def tag_list_to_dict(cls, tag_list):
```

Comment:
`cls` is not used, this shouldn't be a class method, but a normal function - Please use `flake8`, it should warn you about this unused variable

Reply:
We've talked about this and decided not to change it yet. We are running both pylint and flake8 which it passes. We're not running black. See tox.ini for details.

Personally I don't think any of the alternatives are preferable but we can talk about stuff like this in the guild sometime.

Reply:
fair enough, I would have made it a function in the same module as the class that uses it, that way it it can be used outside the module a little easier, but it's a minor thing. perfect guild topic